Platonus Tester/MainWindow.xaml.cs

```diff
@@ -68,7 +68,7 @@ public MainWindow()
                 RBVariant4,
                 RBVariant5,
             };
-            RbVariant1.IsChecked = true;
+            RbVariant1.IsChecked = false;
```

Comment:
That change is unnecessary.

By default, the first option shoul be checked to avoid situations when no one option was checked

```suggestion
            RbVariant1.IsChecked = true;
```